src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlConnectionString.cs

```diff
@@ -1102,20 +1102,6 @@ internal PoolBlockingPeriod ConvertValueToPoolBlockingPeriod()
             }
         }
 
-#if NETFRAMEWORK
```

Comment:
This diff is a bit misleading. This "deleted" section simply moved down and the NetLib code moved up so that the NetLib code is available on netcore.